doc/source/build-actions/index.rst

```diff
@@ -0,0 +1,52 @@
+Build actions
+=============
+
+The build actions allow for building artifacts for a Python library. These
+artifacts include both source distribution files and wheels.
+
+
+Build library action
+--------------------
+This action allows for building source and wheel artifacts for a Python library.
+
+.. jinja:: build-library
+
+    {{ inputs_table }}
+
+
+Code sample for using this action:
+
+.. code-block:: yaml
+
+    build-library:
+      name: Build library
+      runs-on: ubuntu-latest
+      steps:
+        - name: "Build library source and wheel artifacts"
+          uses: pyansys/actions/build-library@main
+          with:
+            library-name: "ansys-<product>-<library"
+
+
+Build C-extension library action
+--------------------------------
+This action allows for building wheel artifacts for a Python library using
```

Comment:
```suggestion
This action builds wheel artifacts for a Python library using
```
Wondering if C-extension is a name that should be in a literal tag or perhaps be a link?

Reply:
I was looking in the [C extension official documentation](https://docs.python.org/3/extending/extending.html) an noticed that:

* It should read `C extension` and not `C-extension`

Does this affect the usage of literal tag?